src/views/Error.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="container">
+    <!-- 404 error starts -->
+    <div v-if="isPageNotFound">
+      <h1>404</h1>
+      <p>You have visited a wrong link</p>
+      <p>आप गलत लिंक पर आ गए हैं</p>
+    </div>
+    <!-- 404 error ends -->
+
+    <!-- Browser error starts -->
+    <div v-if="isBrowserError">
+      <!-- failsafe begins -->
+      <div v-if="hasFailSafe">
+
+        <!-- failsafe G-form begins -->
+        <div v-if="isFailSafeGform">
+          <div class="lead_text">
+            <p>इसी प्रतियोगिता में Google फॉर्म से हिस्सा लेने के लिए नीचे बटन पे क्लिक करें</p>
+            <a
+              :href=value.failsafe_url
+              class="icon-block">
+              <img src="../assets/google_form.svg">
+            </a>
+            <hr class="solid">
+          </div>
+        </div>
+        <!-- failsafe G-form ends -->
+
+      </div>
+      <!-- failsafe ends -->
+
+      <div class="lead_text" >
+        <p>यह वेबसाइट सिर्फ Google Chrome पे चलेगी </p>
+        <p>This website will only work on Google Chrome</p>
+      </div>
+      <a 
+        href="https://play.google.com/store/apps/details?id=com.android.chrome"
+        class="icon-block">
+        <img src="../assets/chrome.svg">
+      </a>
+      <div class="lead_text" style="font-size:1em">
+        <p>ऊपर click करके Google Chrome download करें</p>
+        <p>या फिर</p>
+        <p>Link कॉपी करके Google Chrome में खोलें</p>
+      </div>
+
+    </div>
+    <!-- Browser error ends -->
+
+  </div>
+</template>
+
+<script>
+export default {
+  name: "PageNotFound",
+  props: ['type', 'value'],
+  
+  created() {
+    document.getElementById('nav').style.display = "none";
+  },
+
+  computed: {
+    isPageNotFound() {
```

Comment:
To consider: this can all be done with one method and an "errortype" and then with the error type, you can display the correct page